docs/configuration.md

```diff
@@ -169,5 +169,108 @@ php artisan up
 ```
 > {tip} You may customize the default maintenance mode template by defining your own template at `resources/views/errors/503.blade.php`.
 
+## Configuration options
+
+|Name|Description|Default|
+|---|---|---|
+|`APP_NAME`|The gallery name|`Lychee`|
+|`APP_ENV`|Environment of your gallery, `production` or `development`|`production`|
+|`APP_DEBUG`|Enable debug mode|`false`|
+|`APP_URL`|The URL of your gallery|`http://localhost`|
+|`APP_KEY`|Your app key which is used for encryption (set during installation)|`null`|
+|`TIMEZONE`|The timezone of your server|system timezone of server|
+|`DEBUGBAR_ENABLED`|Enable debugbar (only for debugging)|`false`|
+|`LOG_CHANNEL`|Default log channel, `single`, `daily`, `slack`, `syslog`, `errorlog`, `monolog`, `custom` or `stack`|`stack`|
+|`DB_CONNECTION`|Database type, `mysql`, `pgsql`, `sqlite` or `sqlsrv` (`sqlsrv` is not supported officially and may not work)|`mysql`|
+|`DB_DATABASE`|Path to SQLite database or name of MySQL/PostgreSQL DB|`database/database.sqlite` or `forge`|
+|`DB_HOST`|Host of DB server|`127.0.0.1`|
+|`DB_PORT`|Post of DB server|`3306` (MySQL) or `5432` (PostgreSQL)|
+|`DB_USERNAME`|Username of database|`forge`|
+|`DB_PASSWORD`|Password of database user|empty string|
+|`DB_OLD_LYCHEE_PREFIX`|Table prefix of old Lychee database|empty string|
+|`DB_LOG_SQL`|Log SQL statements (only use for debugging)|`false`|
+|`BROADCAST_DRIVER`|The broadcaster used if the framework needs to broadcast an event, `pusher`, `redis`, `log` or `null`|`null`|
+|`PUSHER_APP_KEY`|App key for the `pusher`|`null`|
+|`PUSHER_APP_SECRET`|App secret for the `pusher`|`null`|
+|`PUSHER_APP_ID`|App ID for the `pusher`|`null`|
+|`PUSHER_APP_CLUSTER`|App cluster for the `pusher`|`null`|
+|`CACHE_DRIVER`|The driver used for caching, `apc`, `array`, `database`, `file`, `memcached`, `redis` or `dynamodb`|`file`|
+|`REDIS_HOST`|Redis host|`127.0.0.1`|
+|`REDIS_PASSWORD`|Redis password|`null`|
+|`REDIS_PORT`|Redis port|`6379`|
+|`LYCHEE_UPLOADS`|Path to uploads directory|`uploads/`|
+|`LYCHEE_UPLOADS_URL`|URL to uploads directory|`uploads/`|
+|`LYCHEE_DIST`|Path to dist directory|`dist/`|
+|`LYCHEE_DIST_URL`|URL to dist directory|`dist/`|
+|`MAIL_DRIVER`|Mailer type, `smtp`, `ses`, `mailgun`, `postmark`, `sendmail`, `log` or `array`|`smtp`|
+|`MAIL_HOST`|Host of SMTP server|`smtp.mailgun.org`|
+|`MAIL_PORT`|Port of SMTP server|`587`|
+|`MAIL_ENCRYPTION`|Encryption for SMTP server|`tls`|
+|`MAIL_USERNAME`|Username of SMTP server|`null`|
+|`MAIL_PASSWORD`|Password of SMTP server|`null`|
+|`MAIL_FROM_ADDRESS`|"From" address|`hello@example.com`|
+|`MAIL_FROM_ADDRESS`|"From" name|`Example`|
+|`QUEUE_CONNECTION`|Connection for queues, `sync`, `database`, `beanstalkd`, `sqs` or `redis`|`sync`|
+|`SESSION_DRIVER`|Driver for sessions, `file`, `cookie`, `database`, `apc`, `memcached`, `redis`, `dynamodb` or `array`|`file`|
+|`SESSION_LIFETIME`|Session lifetime in minutes|`120`|
```

Comment:
Maybe add a bit more of an explanation about this one? Say, "Idle session timeout in minutes; the session will need to be reinitialized once it expires"?